src/actions/search-results.js

```diff
@@ -125,9 +125,7 @@ export function mixDataInput () {
       const items = result.graphql.items;
 
       items.forEach(item => {
-        if (item.tile) {
-          mixture.push(item);
-        } else if (item.packageOffer) {
```

Comment:
There were two clauses in the conditional which performed `mixture.push(item)`, so have that as the default and handle packages separately.